subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
                 </a>
             </div></li>
             <li><a href="/userguide/getting_started.html">Getting Started</a></li>
-        </ul>
-
-        <h3 id="using-gradle">Using Gradle</h3>
-        <ul>
-            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+            <li><a href="/userguide/installation.html">Installation</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading &amp; Migrating</a>
                 <ul id="upgrading-gradle">
-                    <li><a href="/userguide/upgrading_version_4.html">version 4.x to 5.0</a></li>
-                </ul>
-            </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#migrating-to-gradle" aria-expanded="false" aria-controls="migrating-to-gradle">Migrating to Gradle</a>
-                <ul id="migrating-to-gradle">
+                    <li><a href="/userguide/upgrading_version_4.html">Upgrading from Gradle 4.x to 5.0</a></li>
                     <li><a href="https://guides.gradle.org/migrating-from-maven/">Migrating From Maven</a></li>
                 </ul>
             </li>
-            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Getting Started Guides</a></li>
-            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
+            <li id="tutorials"><a href="https://guides.gradle.org/#getting-started">Tutorials</a></li>
```

Comment:
The name "Getting Started Guides" is longer and doesn't provide any more information than "Tutorials", while being longer. I find the name change particularly awkward with the new "Getting Started" link in the navigation. It feels a little dissonant to me.

Reply:
Makes sense.